Resources/Prototypes/_NF/Entities/Objects/Specific/Engineering/construction_bag.yml

```diff
@@ -0,0 +1,73 @@
+- type: entity
+  parent: BaseStorageItem
+  id: ConstructionBag
+  name: construction bag
+  description: A robust bag for construction specialists and engineers alike to carry large amounts of construction materials. Magnetises any nearby construction materials when attached to a belt.
+  components:
+  - type: MagnetPickup
+  - type: Sprite
+    sprite: _NF/Objects/Specific/Engineering/construction_bag.rsi
+    state: icon
+  - type: Clothing
+    sprite: _NF/Objects/Specific/Engineering/construction_bag.rsi
+    quickEquip: false
+    slots:
+    - belt
+  - type: Item
+    size: Ginormous
+  - type: Storage
+    maxItemSize: Normal
+    grid:
+    - 0,0,9,3
+    quickInsert: true
+    areaInsert: true
+    whitelist:
+      tags:
+      - Ingot
+      - RawMaterial
+      - RodMetal1
+      - Sheet
+      components:
+      - RodMetal   
```

Comment:
Got this to magnet up cotton, pyrotton, and prize tickets, which seemed odd.
Cables don't seem to get picked up, which I might've expected.

Reply:
And fixed this one too

Reply:
oh, wait, prize tickets..

Reply:
![image](https://github.com/new-frontiers-14/frontier-station-14/assets/65374927/57d87123-53f3-47fa-a939-e910b39a54e2)

Reply:
Done

Reply:
Retesting.